janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/TransactionConfiguration.java

```diff
@@ -199,6 +199,12 @@ public interface TransactionConfiguration extends BaseTransactionConfig {
      */
     boolean isSkipDBCacheRead();
 
+    /**
+     * Returns true if all properties and edges are lazy-loaded: ids and values are deserialized upon demand.
+     * When enabled, it can have a boost on large scale read operations, when only certain type of relations are being read.
+     */
+    boolean isLazyLoadRelations();
+
```

Comment:
It would be good to add this parameter description to the documentation as well (here: https://github.com/JanusGraph/janusgraph/blob/master/docs/basics/transactions.md#transaction-configuration ).

Reply:
Done